### PR TITLE
replace travis-ci by GitHub Actions

### PR DESCRIPTION
**Pull request by XenoAmess**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
```
Comment (jamietanna) [inline .github/dependabot.yml:3]: should this be maven?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      - uses: actions/cache@v2.1.6
+        with:
+          path: ~/.gradle/caches/
+          key: ${{ runner.os }}-gradle-caches-${{ hashFiles('**/build.gradle') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - uses: actions/cache@v2.1.6
+        with:
+          path: ~/.gradle/wrapper/
+          key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('**/build.gradle') }}
```
Comment (jamietanna) [inline .github/workflows/build.yml:32]: should these be related to Maven, not Gradle?

File: .github/dependabot.yml (hunk 1 above)
Comment (XenoAmess) [reply .github/dependabot.yml:3]: > should this be maven?

looks like I made a mistake.
wait me for 20 minutes.

Comment (XenoAmess) [reply .github/dependabot.yml]: @jamietanna done.

**Comment by XenoAmess** [comment]
@jamietanna sorry for making that stupid mistake.
I copied one of my gradle project's ci to you instead a maven one.
fix done.

**Pull request merged**